cinn/backends/ir_schedule_test.cc

```diff
@@ -2497,6 +2498,70 @@ void test_compute_inline4(const float* __restrict__ A, float* __restrict__ C)
 }
 #endif
 
+TEST(IrSchedule, reverse_compute_inline1) {
+  Context::Global().ResetNameId();
+  Expr M(32);
+  Expr N(32);
+  Expr P(32);
+
+  Target target = common::DefaultHostTarget();
+
+  Placeholder<float> A("A", {M, N, P});
+  auto B = Compute(
+      {M, N, P}, [&](Var i, Var j, Var k) { return A(i, j, k) + Expr(1.f); }, "B");
+  auto C = Compute(
+      {M, N, P}, [&](Var i, Var j, Var k) { return B(j, i, k) * Expr(2.f); }, "C");
+
```

Comment:
参考如下示例构造测试case：https://github.com/PaddlePaddle/community/blob/master/pfcc/call-for-contributions/CINN/CINN_ir_schedule.md#5reversecomputeinline。
验证能否产出如文档所示的效果。
```suggestion
  Expr M(32);
  Expr N(64);

  Target target = common::DefaultHostTarget();

  Placeholder<float> A("A", {M, N, P});
  auto B = Compute(
      {M, N}, [&](Var i, Var j) { return Expr(1.f) + A(i, j); }, "B");
  auto C = Compute(
      {N, M}, [&](Var i, Var j) { return Expr(2.f) * B(j, i);}, "C");

```

Reply:
好的，这里我已经改成了文档中的形式